### PR TITLE
Add className option for toolbar items

### PR DESCRIPTION
**Pull request by mkaz**
Allows for passing in a className when registering a toolbar button to allow to target via CSS.

An example:
```js
   Prism.plugins.toolbar.registerButton( 'my-button', {
        text: 'My Button',
        className: 'my-button',
    });
```
This will add the span element `<span class="my-button">My Button</span>`

### CODE REVIEW

**Comment by mkaz** [comment]
@RunDevelopment Updated my PR per review, thanks!

File: plugins/toolbar/prism-toolbar.js [hunk 1]
```diff
@@ -14,6 +14,7 @@
 	 * @property {string} text The text displayed.
 	 * @property {string} [url] The URL of the link which will be created.
 	 * @property {Function} [onClick] The event listener for the `click` event of the created button.
+	 * @property {string} className The class attribute to include with element.
```
Comment (mAAdhaTTah) [inline plugins/toolbar/prism-toolbar.js]: Mark as optional:

```suggestion
	 * @property {string} [className] The class attribute to include with element.
```

**Comment by mAAdhaTTah** [comment]
One minor nit but lgtm!

**Comment by mAAdhaTTah** [comment]
Thank you for the contribution!

**Pull request merged**